docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -590,9 +531,11 @@ it('main thread script', async () => {
 });
 ```
 
-#### 控制渲染所用的线程
+在这个例子中，我们触发了一个 `tap` 事件，并在事件处理器中调用了 `globalThis.cb` 函数，然后断言 `globalThis.cb` 函数被调用了一次，并且事件对象中的 `key` 属性为 `value`。
+
+### 控制渲染所用的线程
 
-`render` 方法的第二个参数中支持 `enableMainThread` 和 `enableBackgroundThread` 两个选项，用于开启主线程和后台线程。默认情况下 `enableMainThread` 为 `false`，`enableBackgroundThread` 为 `true`，这样主线程不会渲染首屏，只有后台线程运行完整的 Preact 运行时，并将 UI 更新发送到主线程进行处理。我们之所以不同时开启主线程和后台线程，是因为单元测试并没有和使用 Rspeedy 构建一样将文件编译成两份产物，这样会导致最顶层的代码只能执行一次，并且默认是在后台线程中。假如你写了一个这样的组件：
+`render` 方法的第二个参数中支持 `enableMainThread` 和 `enableBackgroundThread` 两个选项，用于开启主线程和后台线程。默认情况下 `enableMainThread` 为 `false`，`enableBackgroundThread` 为 `true`，这样主线程不会渲染首屏，只有后台线程运行完整的 Preact 运行时，并将 diff 结果发送到主线程进行协调处理。我们之所以不同时开启主线程和后台线程，是因为单元测试并没有和使用 Rspeedy 构建一样将文件编译成两份产物，这样会导致最顶层的代码只能执行一次，并且默认是在后台线程中。假如你写了一个这样的组件：
```

Comment:
The writing here is a bit hard to follow. Consider starting from the background:

e.g. "在中单元测试中，... 不会将文件编译成两份产物 ..." then "默认情况下 ... enableBackgroundThread true " and finally " render 方法的第二个参数中支持 enableMainThread 选项，用于开启主线程渲染"

Also I can't understand this statement "默认情况下 enableMainThread 为 false，enableBackgroundThread 为 true，这样主线程不会渲染首屏，" are you trying to say we are not testing for IFR?

Reply:
You should also try to make the next paragraph more clearer ...

I understand it's difficult because it relies on the understanding of dual-threaded model. Try to reference to other docs (编程思想、渲染流程、代码拆分）instead of re-explaining things here